modules/data.remote/R/call_MODIS.R

```diff
@@ -14,78 +14,133 @@
 ##' @param band_qc string value for which quality control band, or use "NA" if you do not know or do not need QC information (optional)
 ##' @param band_sd string value for which standard deviation band, or use "NA" if you do not know or do not need StdDev information (optional)
```

Comment:
Throughout this file there are lot's of print statements that they need to be replaced with proper pecan.logger functions.

Reply:
Agreed